LearningMissionSimulation/PlaygroundGavril.cs

```diff
@@ -237,7 +262,41 @@ public void CreateClassrooms(int classroomCount)
 
         public void AssignInstructorsToClassrooms()
         {
-            throw new NotImplementedException();
+
+            string action = "Assign Instructors To Classrooms";
+            ReportHeader(actionName: action);
+
+            if (classroomList.Count == 0)
+            {
+                ReportError(missingResource: "Classroom", failedAction: action);
+            }
+            else
+            {
+                int count = 0;
+                foreach (var classroom in classroomList)
+                {
+                    List<Instructor> instructorList;
+                    if (ModuleInstructorDictionary.TryGetValue(classroom.Module.Id, out instructorList))
+                    {
+                        if (instructorList.Count > 0)
+                        {
+                             classroom.Head = instructorList[AttributeGenerator.random.Next(0, instructorList.Count)];
+                             count++;
+                        }
+                        else
+                        {
+                            ReportError(missingResource: "Instructor", failedAction: action);
+                        }
+                    }
+                    else
+                    {
+                        ReportError(missingResource: "Instructor", failedAction: action);
+                    }                   
+                }
+                ReportSummary(actionName: action, itemCount: count);
```

Comment:
Move ReportSummary to the bottom of the method (where it was before).

Reply:
Fixed The Issue

Reply:
You did NOT fix it, please see my comments above and fix.

Reply:
I Fixed Issues - 1